tests/test_list

```diff
@@ -4,3 +4,4 @@ test_iperf3.sh
 test_sysbench.sh
 test_uperf.sh
 test_byowl.sh
+test_ycsb.sh
```

Comment:
Should the testname reflect the database?

Reply:
We're not testing the database at all here, all the test tries to see if ycsb can actually run. For sanity purposes it'll load the database.